### PR TITLE
Update Twenty Twenty-One theme references

### PR DESCRIPTION
**Pull request by jffng**
👋 

We're updating theme name for the block-based version of Twenty Twenty-One: https://github.com/WordPress/theme-experiments/pull/145

This PR updates the references to match the updated theme slug once that PR lands. I may have missed some references so it would be great to get some eyes from folks who are more familiar with the tests.

### CODE REVIEW

**Comment by talldan** [comment]
I think we'd want to merge both the theme-experiments PR and this very soon after one another to avoid things breaking here.

Someone with admin privileges might need to merge this one, as it seems like it wouldn't be possible to get the tests passing here (without merging the other PR, then re-running them, then merging).

**Review by ntsekouras — APPROVED** [review]
LGTM 👍 - let's merge right after the change of tt1.

**Pull request merged**
